libc/src/__support/FPUtil/generic/FMA.h

```diff
@@ -168,17 +168,12 @@ fma(InType x, InType y, InType z) {
   //      z :    10aa...a
   // - prod :     1bb...bb....b
   // In that case, in order to store the exact result, we need at least
-  //   (Length of prod) - (MantissaLength of z) = 2*(52 + 1) - 52 = 54.
-  // TODO:                            53? (Explicit mantissa.) ^
+  //   (Length of prod) - (Fraction length of z) = 2*(52 + 1) - 52 = 54.
```

Comment:
Not sure if the explanation of `- 52` is right.

Reply:
please update the comments to reflect the generic implementation.